bootstrap/postgres/configs.go

```diff
@@ -451,7 +464,36 @@ func (cr configRepository) RemoveChannel(ctx context.Context, id string) error {
 	return nil
 }
 
+func (cr configRepository) ConnectThing(ctx context.Context, channelID, thingID string) error {
+	if thingID == "" || channelID == "" {
+		return repoerr.ErrMalformedEntity
+	}
+
+	qCheck := `SELECT EXISTS(SELECT 1 FROM configs WHERE magistrala_thing = $1), state FROM configs WHERE magistrala_thing = $1`
+	var exists bool
+	var state bootstrap.State
+	if err := cr.db.QueryRowxContext(ctx, qCheck, thingID).Scan(&exists, &state); err != nil {
+		return repoerr.ErrNotFound
+	}
+	if !exists {
+		return repoerr.ErrNotFound
+	}
+	if state == bootstrap.Active {
+		return repoerr.ErrConflict
+	}
+
+	q := `UPDATE configs SET state = $1 WHERE EXISTS (
+		SELECT 1 FROM connections WHERE config_id = $2 AND channel_id = $3)`
+	if _, err := cr.db.ExecContext(ctx, q, bootstrap.Active, thingID, channelID); err != nil {
+		return errors.Wrap(errConnectThing, err)
+	}
+	return nil
+}
+
 func (cr configRepository) DisconnectThing(ctx context.Context, channelID, thingID string) error {
+	if thingID == "" || channelID == "" {
+		return repoerr.ErrMalformedEntity
+	}
```

Comment:
@JeffMboya 
What happens if `thingID` and `channelID` are empty?  
Did postgres throws any error or its doing something else wrong?

Reply:
@arvindh123 I don't see any errors from postgress if `thingID `and `channelID` are empty